tools/idr_download/idr_download_by_ids.xml

```diff
@@ -9,44 +9,51 @@
         <requirement type="package" version="3.1.3">matplotlib</requirement>
     </requirements>
     <command><![CDATA[
-        #if str($id_source.source) == 'link':
-            python -c 'print("${id_source.image_id_link}".split("?show=")[-1].replace("|", "\n"))'
+        mkdir downloads && cd downloads &&
+        #if str($image_ids.source) == 'link':
+            python -c 'print("${image_ids.id_spec}".replace(",", "|").split("?show=")[-1].replace("|", "\n"))'
             ## https://idr.openmicroscopy.org/webclient/?show=image-3426274|image-3426275|image-3426276|image-3426277
         #else:
-            cat ${id_source.image_id_data}
+            cat ${image_ids.id_spec}
         #end if
         | python '$__tool_directory__/idr_download_by_ids.py' 
-            -c $channel 
-            -f $frame
-            -z $z_section
-            --center $x_coord $y_coord $width $height
-            &> $out_log
+        #set $channel = str($channel).strip()
+        #if $channel:
+            -c $channel
+        #end if
+        -f $frame
+        -z $z_section
+        --center $x_coord $y_coord $width $height
 
```

Comment:
should `--rectangle` also be implemented?

Reply:
rectangle is currently implemented so it should probably be named ``--rectangle``

Reply:
For this round, having only support for ``rectangle`` will be enough